### ISSUE TITLE
Update to 1.12.2

### ISSUE THREAD

**Issue by realKSMC** [GameModsBR/VoiceChat#19]
## Update request

Could you please update the VoiceChat Mod to 1.12.2 or minimum 1.10.2?
It is just outdated actually :/

**Comment by utkabobr** [comment]
Did you tried using VoiceChat for 1.12?

**Comment by utkabobr** [comment]
For 1.10.2: https://github.com/GameModsBR/VoiceChat/tree/1.10.2

**Comment by realKSMC** [comment]
Oh didn't noticed, thx alot.
Can you give me an advice to compile it?

**Comment by realKSMC** [comment]
Found out how :)

**Comment by realKSMC** [comment]
Yes, the current build is for version 1.12, but the question was about 1.12.2 :) Actually the version for 1.12 does not crash minecraft, but you cannot hear each other like I commented in another question. Pls update :)

**Comment by realKSMC** [comment]
Sound works with NetworkType 0 :)

**Comment by utkabobr** [comment]
NetworkType 1 is the UDP port on your minecraft server's port (Don't work if query is enabled and port not changed)
